### PR TITLE
faq: modification of the file update question

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

**Comment by krzysztof** [comment]
This PR modifies the FAQ question about file replacement, which mentions versioning as the default update solution.

question_faq
layout_faq

File: content/help/contents.lr [hunk 1]
```diff
@@ -15,13 +15,21 @@ question: Can I change the file in the already published record?
...
 
-Once the record has been published, you can no longer change the files. Exceptionally, we allow small modifications to a record's files, if and only if the record was published recently (less than a month). If you have spotted mistakes such as typos, accidental omission of important files/inclusion of hidden files or confidential files and would like to update them, please contact us and include the following information:
-
-* Send us the email from the address your are registered with in Zenodo.
-* Direct link to the record you would like to update (e.g. DOI or URL).
-* Mention any use of the DOI in e.g. papers or social media.
-* Include the new files as email attachment or provide the public URL to files if they are too big for email.
-* Provide a reason for update.
+Once the record has been published, you can no longer change the files in the record, nonetheless, there are several options:
+1. If the record has a Zenodo DOI (pre-fixed: ``10.5281``):
```
Comment (lnielsen) [inline content/help/contents.lr]: pre-fixed -> prefix

File: content/help/contents.lr [hunk 2]
```diff
@@ -15,13 +15,21 @@ question: Can I change the file in the already published record?
...
+ * The recommended way is to create a new version of a record (more details availabe [here](/#versioning)).
+   Most of the cases fall under this category, which include modifications of the dataset/software, addition or removal of files, new versions of datasets, software or revisions of the papers, including major revisions as well typos and formatting/styling.
+ * Exceptionally, we allow modifications to a record's files "in-place", if and only if, its DOI has already been cited somewhere by it's **exact** version DOI, and it is no longer possible to update it.
+   A typical case is a dataset cited in a paper, which is already in-print. In those cases you should contact us.
+1. If the record has a custom (non-Zenodo) DOI:
```
Comment (lnielsen) [inline content/help/contents.lr]: If the record has a custom (non-Zenodo) DOI -> 

If the record has a non-Zenodo DOI:

File: content/help/contents.lr [hunk 3]
```diff
@@ -15,13 +15,21 @@ question: Can I change the file in the already published record?
...
-* Send us the email from the address your are registered with in Zenodo.
-* Direct link to the record you would like to update (e.g. DOI or URL).
-* Mention any use of the DOI in e.g. papers or social media.
-* Include the new files as email attachment or provide the public URL to files if they are too big for email.
-* Provide a reason for update.
+Once the record has been published, you can no longer change the files in the record, nonetheless, there are several options:
+1. If the record has a Zenodo DOI (pre-fixed: ``10.5281``):
+ * The recommended way is to create a new version of a record (more details availabe [here](/#versioning)).
+   Most of the cases fall under this category, which include modifications of the dataset/software, addition or removal of files, new versions of datasets, software or revisions of the papers, including major revisions as well typos and formatting/styling.
+ * Exceptionally, we allow modifications to a record's files "in-place", if and only if, its DOI has already been cited somewhere by it's **exact** version DOI, and it is no longer possible to update it.
```
Comment (lnielsen) [inline content/help/contents.lr]: Exceptionally, we allow small modifications to a record's files, if and only if the record was published recently (less than one week). If you have spotted mistakes such as typos, accidental omission of important files/inclusion of hidden files or confidential files and would like to update them, please contact us and include the following information.

**Pull request merged**